jwt/src/main/java/me/uport/sdk/jwt/model/ArbitraryMapSerializer.kt

```diff
@@ -49,25 +38,24 @@ object ArbitraryMapSerializer : KSerializer<Map<String, Any?>> {
         is Array<*> -> JsonArray(this.map { it.toJsonElement() })
         else -> {
             //supporting classes that declare serializers
-            val jsonParser = Json(JsonConfiguration.Stable)
-            val serializer = jsonParser.context.getContextualOrDefault(this)
-            jsonParser.toJson(serializer, this)
+            val jsonParser = Json
+            jsonParser.encodeToJsonElement(this)
```

Comment:
I think I am missing a serializer here. Any ideas what is required? @mirceanis

Reply:
not really :(
I am not familiar with the latest kotlinx.serialization requirements, unfortunately.